about.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: docs
+title: About
+headline: All About Pico
+description: What *is* a "stupidly simple, blazing fast, flat file CMS" anyway?
+toc:
+  a-flat-file-cms: A Flat File CMS
+  web-pages-without-html:
+    _title: Web Pages Without HTML
+    markdown-makes-formatting-easy: Markdown Makes Formatting Easy
+    control-your-metadata-with-yaml: Control Your Metadata With YAML
+  flexibility-and-customization:
+    _title: Flexibility and Customization
+    theming-with-twig-templates: Theming With Twig Templates
+    plugins-extend-picos-php: Plugins Extend Pico's PHP
+  pico-is-open-source: Pico is Open Source
+  getting-started: Getting Started
+nav-url: /about/
+nav: 2
+galleries:
+  standalone:
+    style: magnify
+    images:
+      -
+        heading: Pico's Default Theme
+        description: It's a bit bare... but that's intentional!
+        thumbnail: /style/images/docs/about/thumbnails/default_theme.png
+        fullsize: /style/images/docs/about/fullsize/default_theme.png
+        styles: "float: right; margin-left: 2em; border: 1px solid #CCC; border-top: none;"
+  workflow:
+    headline: Pico's Workflow
+    description: Creating content with Pico couldn't be simpler!
+    style: carousel-box
+    images:
+      -
+        heading: Pico's Easy to Install!
```

Comment:
I know that this (probably, I'm not sure :smile:) is grammatically correct... However, "Pico's" doesn't "feel" like "Pico is" for me in the first place... It's a bit odd to read :unamused: